### ISSUE TITLE
Leaking route and area information of secret crags

### ISSUE THREAD

**Issue by killakalle** [theCrag/website#4179]
I've been logging many ascents in my "hidden" (secret) gym.

The routes, that are otherwise not visible to the public, do now show up in my private profile

<img width="711" alt="image" src="https://user-images.githubusercontent.com/30682796/219975346-4811e74d-e9cf-4a11-8981-f9adbbe926f7.png">

Profile:
https://www.thecrag.com/climber/killakalle

Expected:
- Routes in hidden crags don't show up in the stats in the profile

**Comment by scd** [comment]
Yup this looks like a bit of an issue. Probably not too hard to fix. We will have to talk about the rules internally, but I expect it will go something like:

 * You should see your own ticks from hidden crags.
 * You should not see ticks from hidden crags of somebody else's profile even if they are following you
 * If the ticks are excluded then they are excluded everywhere in your profile (ie not in the charts).
